DVMultiplayer/Patches/PassengerJobs/PassengerJobsGeneratorPatch.cs

```diff
@@ -1,44 +0,0 @@
-﻿using HarmonyLib;
```

Comment:
Just FYI: there's a `git mv` command; you don't have to manually move (rename) then `git rm` and `git add`.